Header header
float32 distance_cm
